source/content/nutzen/api-nutzen.rst

```diff
@@ -1,154 +1,473 @@
+          	'it': 'description test_API_POST package it'
+          	},
+        'title': {
+          	'fr': 'title test_API_POST package fr', 
+          	'de': 'title test_API_POST package de', 
+          	'en': 'title test_API_POST package en', 
+          	'it': 'title test_API_POST package it'
+          	}, 
+        'publisher': {
+            'url': 'YOUR_ORGANIZATION_S_URL',
+            'name': 'YOUR_ORGANIZATION'
+            },
+        'keywords': {
+            'fr': ['list', 'of','keywords', 'per','language','fr'], 
+            'de': ['list', 'of','keywords', 'per','language','de'],
+            'en': ['list', 'of','keywords', 'per','language','en'],
+            'it': ['list', 'of','keywords', 'per','language','it'],
+            },
+        'contact_points': [{
+            'email': 'CONTACT.POINT@EMAIL.ADDRESS',
+            'name': 'NAME' 
+            }],
+        'accrual_periodicity': 'http://publications.europa.eu/resource/authority/frequency/MONTHLY',
+        'resources': [{
+        'owner_org': 'YOUR_ORGANIZATION_ID_ON_OPENDATA.SWISS', 
+          	'display_name': {
+          	'issued': '2023-02-12T00:00:00', 
+          		'de': 'display_name test_API_POST resource de', 
+          		'fr': 'display_name test_API_POST resource fr', 
+          		'it': 'display_name test_API_POST resource it'
+          		'en': 'display_name test_API_POST resource en', 
+          	'title': {
+          		}, 
+          		'de': 'title test_API_POST resource de', 
+          		'fr': 'title test_API_POST resource fr', 
+          		'it': 'title test_API_POST resource it'
+          		'en': 'title test_API_POST resource en', 
+          	'download_url': 'https://freetestdata.com/wp-content/uploads/2021/09/Free_Test_Data_200KB_CSV-1.csv',
+          		}, 
+          		'fr': 'description test_API_POST resource fr', 
+          	'description': {
+          		'de': 'description test_API_POST resource de', 
+          		'en': 'description test_API_POST resource en', 
+          		}, 
+          		'it': 'description test_API_POST resource it'
+          	'name': {
+            'format' : 'CSV',
+          		'de': 'name test_API_POST resource de', 
+          		'fr': 'name test_API_POST resource fr', 
+          		'en': 'name test_API_POST resource en', 
+          		'it': 'name test_API_POST resource it'
+          		}, 
+          	'rights': 'NonCommercialAllowed-CommercialAllowed-ReferenceNotRequired', 
+          	'url': 'https://freetestdata.com/wp-content/uploads/2021/09/Free_Test_Data_200KB_CSV-1.csv', 
+            'identifier': 'identifier_resource_for_name_package_''@YOUR-ORGANIZATION-SLUG',
+          	}], 
+        }
+
+    Bitte beachten Sie, dass die Nutzungsbedingungen auf Ebene der Ressource dem kontrollierten Vokabular entsprechen. Vgl. Sie dazu den `DCAT AP CH Standard <https://handbook.opendata.swiss/de/content/glossar/bibliothek/dcat-ap-ch.html?dcat-dataset-accrual-periodicity>`__.
```

Comment:
link should be like this: `https://handbook.opendata.swiss/de/content/glossar/bibliothek/dcat-ap-ch.html#dcat-dataset-accrual-periodicity` 

If you'd like to point to some place on the web-page, you need to use "#"